content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -173,21 +173,21 @@ kubectl apply -f https://k8s.io/examples/pods/probe/http-liveness.yaml
 ```
 
 After 10 seconds, view Pod events to verify that liveness probes have failed and
-the Container has been restarted:
+the container has been restarted:
```

Comment:
Not sure we want to change line 133, 137, 139, 140, 149, 165, 167, 176 ... because we are talking about the Container resource object in this context. Not a strong opinion though.

Reply:
On line 176, the thing that gets restarted is a container and it happens also to be known as a Container in parts of the Kubernetes API.

I tend to lowercase “container” where feasible because:
1. I don't want readers to try making a Container manifest when they should be making a Pod 
manifest.
2. It's a term widely used in IT outside Kubernetes.
3. If a localization team uses the localized term instead, that should be OK.